libse/StripableText.cs

```diff
@@ -215,7 +215,7 @@ public void FixCasing(List<string> namesEtc, bool changeNameCases, bool makeUppe
             if (makeUppercaseAfterBreak && StrippedText.Contains(new[] { '.', '!', '?', ':', ';', ')', ']', '}', '(', '[', '{' }))
```

Comment:
could you also make this a private static var (as you did earlier for another new array)?